security/s2a/internal/conn/expander.go

```diff
@@ -0,0 +1,10 @@
+package conn
+
+import "hash"
+
+// HKDFExpander is the interface for an HMAC-based Extract-and-Expand Key
```

Comment:
Please replace `an HMAC-based Extract-and-Expand Key Derivation Function (HKDF) expander` with `the HKDF expansion function; see https://tools.ietf.org/html/rfc5869 for details.

(We want to make it clear that we are using **the** HKDF function as specified in RFC 5869, as opposed to some HMAC-based Extract-and-Expand key derivation function, which could be ambiguous.)

Reply:
Done